pkg/controller/registry/resolver/rbac.go

```diff
@@ -116,7 +116,11 @@ func RBACForClusterServiceVersion(csv *v1alpha1.ClusterServiceVersion) (map[stri
 	for _, permission := range strategyDetailsDeployment.ClusterPermissions {
 		// Create ServiceAccount if necessary
 		if _, ok := permissions[permission.ServiceAccountName]; !ok {
-			serviceAccount := &corev1.ServiceAccount{}
+			serviceAccount := &corev1.ServiceAccount{
```

Comment:
Can remove these labels if they don't end up getting used.

Reply:
Ended up handling this at the owner reference level instead of with owner labels.